introduction.md

```diff
@@ -12,3 +12,8 @@ ajsg asjg
 ajg
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?